projects/cm_mcu/CommandLineTask.c

```diff
@@ -794,7 +800,7 @@ static BaseType_t eeprom_read(int argc, char ** argv)
   uint64_t data = read_eeprom_multi(addr);
   copied += snprintf(m+copied, s-copied,
 		     "Data read from EEPROM block %d: %08x%08x\r\n",
-		     block,data);
```

Comment:
@gwengardner can you look at this and see if the change is correct

Reply:
This change switches the order of the first and second word (reading 2 words from 0x40 prints out 0x44 then 0x40), though I'm not sure why. The original code prints it out correctly (though honestly, I'm not entirely sure why).

Reply:
the problem is that the `sprintf` call has one too few arguments compared to the format string,  and it depends on the way the complier interprets the passed arguments...

Reply:
Ohh I see. I can't figure out why this change isn't working, but I could change it instead to do two single-word reads instead of splitting up that darn uint64_t. Does that sound okay?

Reply:
that is fine, or, alternatively, you could spend some QT with the debugger. I'd suggest making two `uint32_t` variables explictly and see what they are (rather than doing it in the line with the `snprintf`.

Please push your eventual change to this branch.